### DISCUSSION TITLE
ForceInheritance::run() in Docs?

### DISCUSSION THREAD

**Discussion by NiklasBr** [pimcore/pimcore#13025] [Q&A / Support]
What is this supposed to do? No such class exists in `pimcore/pimcore`

https://github.com/pimcore/pimcore/blob/b675696e8c3dff29e51d97e67fc79d3003c9ca3a/doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/30_Link_Generator.md?plain=1#L79-L91

**Comment by dvesh3** [comment]
please see https://github.com/pimcore/demo/blob/1afa9f9c2d88e7f437042f96a2f595489ac98615/src/Website/Tool/ForceInheritance.php#L26

**Reply by NiklasBr** [reply]
Why is it not in this repository?

**Reply by dvesh3** [reply]
because it is not used internally anywhere in the Core module. It is part of the demo.

**Reply by NiklasBr** [reply]
If it is a requirement for Link Generations I don't understand why it is not fully documented or made available.

**Reply by dvesh3** [reply]
It is also done at some places in advanced object search bundle and DataObject service so maybe we can move it to `Pimcore\Tool` helper.

**Reply by dvesh3** [reply]
Created an issue https://github.com/pimcore/pimcore/issues/13031

**Reply by dvesh3** [reply]
@NiklasBr I closed the above issue as we already have one planned for v11. https://github.com/pimcore/pimcore/issues/12242

**Reply by NiklasBr** [reply]
Ok!
